docker/README.md

```diff
@@ -53,6 +52,27 @@ In order to setup an application service, simply create an ``appservices``
 directory in the data volume and write the application service Yaml
 configuration file there. Multiple application services are supported.
 
+## TLS certificates
+
+Synapse requires a valid TLS certificate. You can do one of:
+
+ * Provide your own certificate and key (as
+   `${DATA_PATH}/${SYNAPSE_SERVER_NAME}.crt|key`, or elsewhere by providing
```

Comment:
Is `|key` supposed to be an actual  key? If so, then should it be `|${key}`?

Reply:
clarified, I hope.

Reply:
Much, thank you :)